lib/services/jira/jira_wiki_converter.rb

```diff
@@ -0,0 +1,322 @@
+require "reverse_markdown"
```

Comment:
Is this running against the existing test suite, or do we have other tests specifically for this converter? I'd like to be sure this is tested before merging, since it would be very easy to cause regressions.